src/main/kotlin/step2/calculator/InputValidator.kt

```diff
@@ -0,0 +1,22 @@
+package step2.calculator
+
+object InputValidator {
+    fun validate(input: String?) {
+        validateInputIsNullOrBlank(input)
+        validateIsArithmeticSymbol(input!!)
+    }
+
+    private fun validateInputIsNullOrBlank(input: String?) {
+        require(!input.isNullOrBlank()) { "입력된 값이 Null 이거나 공백입니다." }
+    }
+
+    private fun validateIsArithmeticSymbol(input: String) {
+        require(!filterDigit(input).any { !ARITHMETIC_SYMBOLS.contains(it) }) { "입력된 값에 사칙 연산 기호가 아닌 값이 존재합니다." }
```

Comment:
```suggestion
        require(ARITHMETIC_SYMBOLS.containsAll(filterDigit(input))) { "입력된 값에 사칙 연산 기호가 아닌 값이 존재합니다." }
```

containsAll을 사용하면 더 간결해지지 않을까요?

참고
- [containsAll](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/contains-all.html)